README.md

```diff
@@ -1467,3 +1476,17 @@ The script serves the Swagger editor locally at a URL where your current version
 of the API documentation is visible. Change the docs in `azul/service/app.py`,
 save, and refresh the page and your changes will appear immediately along with
 any warnings or errors they may introduce.
+
+# 13. Onboarding
+ To assist with getting onboarded to the project, review the following documents:
+
+- [Azul Overview](#1-architecture-overview)  
+- [Getting Started](#2-getting-started)
+- [Contributing Guidelines](CONTRIBUTING.rst)
+  
+Request Access to Cloud Infrastructure and Tools:
+  - Slack [#team-boardwalk](https://ucsc-cgl.slack.com/archives/C705Y6G9Z) 
```

Comment:
We do need to instruct who grants access to Slack. I don't actually know.

Reply:
I believe it was my PM that gave me access to the Slack channel. From within the slack application, anyone can add a member, but an admin has to approve of their addition. Currently our PMs, and the office admin assistants have the  power to approve.